backend/app.js

```diff
@@ -13,7 +13,7 @@ const app = express();
 app.use(
   cors({
     // added origin
-    origin: FRONTEND_URL,
+    origin: [FRONTEND_URL, 'http://localhost:3000'],
     credentials: true,
```

Comment:
What is this change?

Reply:
since we are doing an integration test with the real API. The jsdom is our frontend and it's default URL is localhost:3000,  so if we doesn't include it we will get an cors error during integration test..